base/Dockerfile

```diff
@@ -33,6 +33,8 @@ RUN conda install mamba -y --quiet -c conda-forge \
         xarray \
         cartopy \
         ipyleaflet \
+        jupyterlab==3.1.4 \
```

Comment:
do we need this now that it's using the correct base image for building?

Reply:
we don't use the latest one with jupyterlab 3.1.4 as that one is bringing also py 3..9. I moved to the latest image with py3.8 which has jupyterlab 3.0.4

Reply:
ok makes sense